### PR TITLE
Hello Microverse project

### PR DESCRIPTION
**Pull request by bekure27**
add HTML, CSS and test file and style the HTML.
add test file in gitignore list.

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (KDlamini) [inline test.md:1]: Since the `test.md` file is ignored by the `.gitignore` file it shouldn't exist in the repository. Kindly delete it from Github.

**Review by KDlamini — APPROVED** [review]
### Hi @bekure27 🙋‍♀️,  

Excellent work! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 
✔️  GitHub actions and workflow setup correctly
✔️  README is documented
✔️  Correct PR title and description
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (bekure27) [reply test.md:1]: @KDlamini  thanks. for your constructive feedback.

**Review by mukhammadrizooff — APPROVED** [review]
Hi @bekure27, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions

- [ ] [OPTIONAL] Please recreate the `test.md` file and give some value for check the `.gitignore` is working or not.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**